pkg/guacrest/openapi.yaml

```diff
@@ -99,7 +99,127 @@ paths:
           $ref: "#/components/responses/InternalServerError"
         "502":
           $ref: "#/components/responses/BadGateway"
-
+  "/v0/package/{purl}":
+    get:
+      summary: Get purls related to the specific Package URL (PURL)
+      operationId: getPackagePurlsByPurl
+      parameters:
+        - name: purl
+          in: path
+          required: true
+          description: URL-encoded Package URL (PURL)
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A list of PURLs associated with the specified PURL
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                  description: Package URL (PURL)
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+  "/v0/package/{purl}/vulns":
+    get:
+      summary: Get vulnerabilities for a specific Package URL (PURL)
```

Comment:
IMO this endpoint shouldn't return vulnerabilities in the dependencies of the input, because this duplicates functionality and may lead to drift. Finding vulnerabilities in dependencies can be done by composing a dependency retrieval and a vuln search (without dependencies).

Reply:
To solve this issue, the vulnerability retrieval now returns vulns for the given purl. But, if the `includeDependencies` flag is passed in, then it includes the dependencies for the purl as well as its dependencies.